webapp/src/Components/Player/Overview/SideBar/PlayerProfilePicture.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from "react"
+
+import { CardMedia, createStyles, Tooltip, WithStyles, withStyles } from "@material-ui/core"
+
+interface OwnProps {
+    image: string
+    groups: number[]
+}
+
+type Props = OwnProps
+    & WithStyles<typeof styles>
+
+class PlayerProfilePictureComponent extends React.PureComponent<Props> {
+    public render() {
+        const {image, classes, groups} = this.props
+
+        const colors = {
+            1: "red", // admin
+            2: "#FFC107", // alpha
+            3: "#8BC34A", // beta
+            4: "#03A9F4" // dev
+        }
+        const letters = {
+            1: "A",
+            2: "α",
+            3: "β",
+            4: "D"
+        }
+        const names = {
+            1: "Admin",
+            2: "Alpha Tester",
+            3: "Beta Tester",
+            4: "Developer"
+        }
+        const hierarchy = [1, 4, 2, 3]
+        let userRole = 0
+        for (const role of hierarchy) {
+            if (groups.indexOf(role) !== -1) {
+                userRole = role
+                break
+            }
+        }
```

Comment:
```javascript
const groups = [1]

const roles = [
  {
    name: "Admin",
    color: "red",
    letter: "A",
    hierarchy: 1
  },
  {
    name: "Developer",
    color: "#03A9F4",
    letter: "D",
    hierarchy: 2
  },
  {
    name: "Alpha Tester",
    color: "#FFC107",
    letter: "α",
    hierarchy: 3
  },
  {
    name: "Beta Tester",
    color: "#8BC34A",
    letter: "β",
    hierarchy: 4
  }
]

const userRole = roles
  .filter(({ hierarchy }) => groups.includes(hierarchy))
  .sort(({ hierarchy: a }, { hierarchy: b }) => a - b)[0]

```

Reply:
Probably doesn't even need `hierarchy`. Could rename that in each of the objects to ID and then assume that hierarchy is determined by the first element of the filtered list

Reply:
```javascript
const roles = [
  {
    id: 1,
    name: "Admin",
    color: "red",
    letter: "A"
  },
  {
    id: 2,
    name: "Developer",
    color: "#03A9F4",
    letter: "D"
  },
  {
    id: 3,
    name: "Alpha Tester",
    color: "#FFC107",
    letter: "α"
  },
  {
    id: 4,
    name: "Beta Tester",
    color: "#8BC34A",
    letter: "β"
  }
]

const userRole = roles.filter(({ id }) => groups.includes(id))[0]
```